client/src/components/Board.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import Ticket from './Ticket'
 
 export function Board() {
 	type Props = {
 		title: string;
+
 	};
 	const Column = ({ title }: Props) => {
 		return (
 			<div className="bg-dark-blue min-h-full">
 				<h3 className="text-green text-center p-2">{title}</h3>
+				<Ticket title="example title" tags="bug" date="todays date"></Ticket>
 			</div>
 		);
 	};
 	return (
 		<div className="grid grid-cols-4 bg-blue gap-6 m-10 p-6 min-h-screen">
-			<Column title="To Do" />
+			<Column title="To Do"></Column>
```

Comment:
Change requested: you can removed the closing column and replace it with `/>` again

Reply:
sure.